client-v2/src/main/java/com/clickhouse/client/api/Client.java

```diff
@@ -1032,9 +1032,10 @@ public boolean ping() {
      */
     public boolean ping(long timeout) {
         if (useNewImplementation) {
-            try (QueryResponse response = query("SELECT 1 FORMAT TabSeparated").get(timeout, TimeUnit.MILLISECONDS)) {
+            try (QueryResponse ignored = query("SELECT 1 FORMAT TabSeparated").get(timeout, TimeUnit.MILLISECONDS)) {
```

Comment:
it is good to keep PR changes as minimal as possible. 
I do not mind about this change, but we do not do like this:
- java has no good way to ignore returned value in this case  
- variable names should be meaningful.